SQL (#1679)
enkidevs/curriculum: sql/ddl/create/create-a-table.md

---
## Content

> When creating a new table there are a couple rules which must be always followed to ensure good performance:

Reviewer: I find this rephrasing more natural 🤷‍♂ :

```suggestion
When creating a new table there are some essential rules to follow to ensure good performance:
```

* each table column has a fixed name
* all data in a column must be of the same type

> On top of the rules mentioned above, you must also decide what data you want to include in your table, what name do you assign to the data (column) and what type it is going to be[1].

Reviewer: ```suggestion
On top of the rules mentioned above, you must also decide what data you want to include in your table, what name do you assign to the data (column) and what type is it going to be[1].
```

> The process of creating a new table is very simple. First, we need to tell the database that we want to create a new table using the `CREATE TABLE` statement followed by the table name. Inside the `CREATE TABLE` statement you start defining your columns together with their types, and every subsequent pair of column-type must be preceded by a comma.

Reviewer: Remove the subjective "simply" and normalize the usage of "we" vs "you":

```suggestion
The process of creating a new table contains two steps. First, you tell the database that you want to create a new table using the `CREATE TABLE` statement followed by the table name. Next, inside the `CREATE TABLE` statement, you start defining your columns together with their types, and every subsequent pair of column-type must be preceded by a comma.
```
